hack/run-wsu-ci-e2e-test.sh

```diff
@@ -3,6 +3,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+SKIP_VM_SETUP=""
```

Comment:
I think, we need to tighten the validation. This is something perhaps to add as a `TODO`.